client/Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM fedora:25
 MAINTAINER Luiz Carvalho <lucarval@redhat.com>
 
+ARG UPDATES
 ARG UPDATES_TESTING
 
-RUN dnf update -y && dnf install -y dnf-plugins-core && \
-    if [ $UPDATES_TESTING ]; then dnf config-manager --set-enable updates-testing; fi && \
-    dnf -y update && \
+RUN if [ $UPDATES ]; then dnf update -y; fi && \
```

Comment:
Hey! This same block structure can be used to install differently named packages for different distros, thus sharing the same Dockerfile :)

Ok, this comment is unrelated to this PR